projects/pedal_board/src/pedal_data_tx.c

```diff
@@ -0,0 +1,54 @@
+#include "pedal_data_tx.h"
+#include "ads1015.h"
+#include "brake_data.h"
+#include "can.h"
+#include "can_msg_defs.h"
+#include "can_transmit.h"
+#include "can_unpack.h"
+#include "event_queue.h"
+#include "fsm.h"
+#include "log.h"
+#include "pedal_events.h"
+#include "soft_timer.h"
+#include "throttle_data.h"
+
+#define TIMER_TIMEOUT_IN_MILLIS 100
+
+int16_t brake_position = INT16_MAX;
+int16_t throttle_position = INT16_MAX;
+
+// pedal callback
+static void prv_pedal_timeout(SoftTimerId timer_id, void *context) {
+  PedalDataStorage *pedal_storage = context;
+
+  get_brake_data(pedal_storage, &brake_position);
+  get_throttle_data(pedal_storage, &throttle_position);
+  // SENDING POSITIONS THROUGH CAN MESSAGES
+  CAN_TRANSMIT_PEDAL_OUTPUT((uint32_t)throttle_position, (uint32_t)brake_position);
+
+  soft_timer_start_millis(TIMER_TIMEOUT_IN_MILLIS, prv_pedal_timeout, context, NULL);
+}
+
+int16_t get_brake_position() {
```

Comment:
no one's using these?

Reply:
don't use these in your test. Mock the source of the data instead. If you don't know how to use mocks, ask me.